x-pack/test_serverless/api_integration/test_suites/common/platform_security/response_headers.ts

```diff
@@ -23,7 +23,7 @@ export default function ({ getService }: FtrProviderContext) {
     const baseCSP = `script-src 'report-sample' 'self'; worker-src 'report-sample' 'self' blob:; style-src 'report-sample' 'self' 'unsafe-inline'; frame-ancestors 'self'`;
     const defaultCOOP = 'same-origin';
     const defaultPermissionsPolicy =
-      'camera=(), display-capture=(), fullscreen=(self), geolocation=(), microphone=(), web-share=()';
+      'camera=(), display-capture=(), fullscreen=(self), geolocation=(), microphone=(), web-share=();report-to=violations-endpoint';
```

Comment:
Prolly doesn't matter, but do we need a space after the semicolon?

Reply:
Alsoi, would you mind filling me in on what this does?  I  get it's matching something in svl, but paint me curious! :)

Reply:
The report-to directive allows us to gather telemetry on CSP and permissions-policy violations.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-to

The semicolon is the delimiter, the space after is not necessary.